Dockerfile

```
@@ -1,14 +1,17 @@
 
-# 3. JAR 파일 복사
 COPY build/libs/app.jar app.jar
 
-# 4. 시간대 설정 (선택 사항: RDS와 시간대를 맞추기 위해 추천)
 ENV TZ=Asia/Seoul
+ENV SPRING_PROFILES_ACTIVE=prod
 
-# 5. 애플리케이션 실행
-ENTRYPOINT ["java", "-jar", "app.jar"]
+ENTRYPOINT ["java",
+  "-XX:+UseContainerSupport",
```

Hadolint (2.14.0)

[error] unexpected '"'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)